systemtest/src/main/java/io/strimzi/systemtest/Resources.java

```diff
@@ -647,9 +656,15 @@ private void waitForDeletion(Kafka kafka) {
         IntStream.rangeClosed(0, kafka.getSpec().getKafka().getReplicas() - 1).forEach(podIndex ->
             waitForPodDeletion(kafka.getMetadata().getName() + "-kafka-" + podIndex));
 
+        // Wait for EO deletion
         client().listPods().stream()
                 .filter(p -> p.getMetadata().getName().contains(kafka.getMetadata().getName() + "-entity-operator"))
                 .forEach(p -> waitForPodDeletion(p.getMetadata().getName()));
+
+        // Wait for Kafka Exporter deletion
```

Comment:
Maybe here would be better use LOGGER instead of commend. 
```suggestion
        LOGGER.info("Wait for Kafka Exporter deletion");
```